mobile/src/navigations/onboarding-navigator/employer-onboarding.tsx

```diff
@@ -0,0 +1,31 @@
+import { createDrawerNavigator } from '@react-navigation/drawer';
+import React from 'react';
+
+import { EmployerOnboardingScreenName } from '~/bundles/common/enums/enums';
+import { type EmployerOnboardingNavigationParameterList } from '~/bundles/common/types/types';
+import { Step1 } from '~/bundles/employer/screens/screens';
+
+import { Header } from './components/components';
+
+const Drawer =
+    createDrawerNavigator<EmployerOnboardingNavigationParameterList>();
+
+const EmployerOnboardingNavigator: React.FC = () => {
+    return (
+        <Drawer.Navigator
+            screenOptions={{
+                headerShown: true,
+                header: ({ navigation }): React.ReactNode => (
+                    <Header navigation={navigation} />
+                ),
+            }}
+        >
+            <Drawer.Screen
+                name={EmployerOnboardingScreenName.STEP1}
+                component={Step1}
+            />
+        </Drawer.Navigator>
+    );
+};
+
```

Comment:
I deleted it, but left the comment '// TODO' in RootStack and left the onboarding-navigator folder with a separate **TalentOnboardingNavigator**, because I still need some kind of **EmployerOnboardingNavigator** or going directly to the **onboarding screen** for employer in the future, right?

Reply:
Yes, I guess it will be separate screen

Reply:
So maybe I should delete TalentOnboardingNavigator and leave only OnboardingNavigator?

Reply:
No, it's mo clear that this screens are about talent user